mongo/test/org/immutables/mongo/fixture/SimpleUpdaterTest.java

```diff
@@ -0,0 +1,153 @@
+package org.immutables.mongo.fixture;
+
+import org.bson.types.ObjectId;
+import org.immutables.mongo.types.Id;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+import static org.immutables.check.Checkers.check;
+
+public class SimpleUpdaterTest {
+
+    @Rule
+    public final MongoContext context = new MongoContext();
+
+    private ItemRepository repository;
+
+    @Before
+    public void setUp() throws Exception {
```

Comment:
Regarding formatting, perhaps enforce it using [checkstyle](http://checkstyle.sourceforge.net/) for maven build ?